frontend/app/protected/template.hbs

```diff
@@ -1 +1 @@
-{{liquid-outlet}}
+{{outlet}}
```

Comment:
Ah, i misunderstood. For liquid fire one has to replace the uppermost outlet, but obviously not all the others. Thanks!